languages/rust/src/lf.rs

```diff
@@ -8,10 +8,15 @@
 //! Flipper "modules" (not to be confused with rust modules).
 
 use libc;
-use libc::{c_void, c_int};
+use libc::{c_void, c_int, c_char};
 use std::ops::Deref;
 use std::ptr;
-use ::{ModuleFFI, _lf_module};
+use std::ffi::CString;
+
+pub type _lf_device = *const c_void;
```

Comment:
I'm wondering whether the `_lf_device` type should just be `c_void` so that in the `lf_invoke` signature it's clearer that it's a pointer by having `lf_invoke(device: *const _lf_device, ...)`

Reply:
That is a good point.